test/e2e/upgrades/BUILD

```diff
@@ -12,43 +12,20 @@ go_library(
     srcs = [
         "apparmor.go",
         "configmaps.go",
-        "daemonsets.go",
-        "deployments.go",
         "etcd.go",
         "horizontal_pod_autoscalers.go",
         "ingress.go",
-        "job.go",
         "mysql.go",
         "persistent_volumes.go",
         "secrets.go",
         "services.go",
-        "statefulset.go",
         "sysctl.go",
         "upgrade.go",
     ],
     tags = ["automanaged"],
     deps = [
-        "//pkg/api/v1/helper:go_default_library",
```

Comment:
@foxish i removed unnecessary dependencies manually. i will continue investigating whether there is an automated way to modify BUILD file.